src/classes/Recipe.js

```diff
@@ -1,50 +1,52 @@
-import ingredientsData from '../data/ingredients'
-import Ingredient from './Ingredient'
+import Ingredient from './Ingredient';
 
 class Recipe {
-  constructor(recipeData) {
-    this.id = recipeData.id
-    this.image = recipeData.image
-    this.recipeIngredients = recipeData.ingredients
-    this.instructions = recipeData.instructions
-    this.name = recipeData.name
-    this.tags = recipeData.tags
-    this.ingredients = this.getAllIngredientsData()
+  constructor(recipeData, ingredientsData) {
```

Comment:
Looks awesome!!! Would love to chat more at stand up on how this was discovered it was needed.